go.mod

```diff
@@ -5,5 +5,5 @@ go 1.13
 require (
 	github.com/ethereum/go-ethereum v1.9.10
 	github.com/ipfs/go-log v1.0.4
-	github.com/keep-network/keep-common v1.2.1-0.20201020114759-19c123cbd4f4
+	github.com/keep-network/keep-common v1.2.1-0.20201116121154-016d77aacdce
```

Comment:
Can we update it to `8af057907255bf852d9b2dca15e9e3d1af4ecc02`? This is https://github.com/keep-network/keep-common/pull/58 merged to `master`.

Reply:
Done.